robozonky-installer/robozonky-installer-panels/src/main/java/com/github/triceo/robozonky/installer/panels/RoboZonkyInstallerListener.java

```diff
@@ -300,10 +311,36 @@ CommandLinePart prepareLogging() {
         }
     }
 
+    void prepareDaemons() {
+        Collection<String> contentC;
```

Comment:
Můžu se zeptat, v jakém jazyce se cítíte nejsilnější? Podle této konstrukce bych tipoval, že jste céčkař.

Není to nic proti ničemu, ale v Javě resp. v _RoboZonky_ udržuji v kódu jedno pravidlo: všechny proměnné jsou `final`, a tedy immutable - pouze jednou zapsatelné. Zvyšuje to čitelnost kódu.

Když to uděláte s `contentC`, zjistíte, že budete muset pro všechny ostatní zápisy použít jiné proměnné. To není na škodu - používat jednu proměnnou pro více účelů je "céčkařina", kterou si v Javě můžeme odpustit, protože o paměť se stará JVM za nás.

Některým proměnným se ale můžete také zcela vyhnout, viz následující komentář.